@shopify/polyfills breaks SSR

## Overview

With `@shopify/polyfills` >= `1.1.9` the following happens:

```

Test suite failed to run
--
  |  
  | Cannot find module 'core-js/stable' from 'base.js'
  |  
  | Require stack:
  | node_modules/@shopify/polyfills/dist/src/base.js
  | app/foundation/App/App.tsx
  | app/foundation/App/index.tsx
  | app/index.tsx
  | server/middleware/auth-redirect-loop/auth-loop-response.tsx
  | server/middleware/auth-redirect-loop/auth-redirect-loop.ts
  | server/middleware/auth-redirect-loop/index.ts
  | server/middleware/index.ts
  | server/tests/utilities/create-context.ts
  | server/tests/utilities/index.ts
  | server/middleware/load-testing/tests/load-testing.test.ts
  |  
  | at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:299:11)
  | at Object.<anonymous> (node_modules/@shopify/polyfills/dist/src/base.js:1:1)

```
...

## Consuming repo

> What repo were you working in when this issue occurred?

Shopify/web

## Labels

- [x] Add the `Type: Bug` label to this issue.

### Area

- [x] Add any relevant `Area: <area>` labels to this issue

### Scope

- Is this issue related to a specific package?

  - [ ] Tag it with the `Package: <package_name>` label.

---

## Checklist

- [ ] Please delete the labels section before submitting your issue
- [ ] I have described this issue in a way that is actionable (if possible)

--- a/packages/polyfills/CHANGELOG.md
+++ b/packages/polyfills/CHANGELOG.md
@@ -5,7 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-<!-- ## [Unreleased] -->
+## [Unreleased]
+
+- Update polyfill base imports. [#1461](https://github.com/Shopify/quilt/pull/1461)
 
 ## [1.1.6] - 2019-01-12
 

--- a/packages/polyfills/src/base.ts
+++ b/packages/polyfills/src/base.ts
@@ -1,2 +1,14 @@
-require('core-js/stable');
+// Cover all standardized ES6 APIs.
+require('core-js/es6');
+// Standard now
+require('core-js/fn/array/includes');
+require('core-js/fn/string/pad-start');
+require('core-js/fn/string/pad-end');
+require('core-js/fn/symbol/async-iterator');
+require('core-js/fn/object/get-own-property-descriptors');
+require('core-js/fn/object/values');
+require('core-js/fn/object/entries');
+require('core-js/fn/promise/finally');
+// Ensure that we polyfill ES6 compat for anything web-related, if it exists.
+require('core-js/web');
 require('regenerator-runtime/runtime');

